docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -37,37 +38,39 @@ Custom Search Attributes of the `Text` type cannot be used in **ORDER BY** claus
 
 ### Partial string match
 
-The `=` operator works like **CONTAINS** to find Workflows with Search Attributes that contain a specific word.
+The `=` operator functions similarly to **CONTAINS**, helping to find Workflows with Search Attributes containing a specific word.
```

Comment:
Because "function" has such a specific meaning in software development, I'm inclined to use "works" in this context. And although "like" provides an informal tone, it is indeed accepted by the Google style guide.